packages/ppx_sexp/ppx_sexp.0.2.2/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "Jonathan Chan <jyc@fastmail.fm>"
+authors: "Jonathan Chan <jyc@fastmail.fm>"
+homepage: "https://bitbucket.org/jyc/ppx_sexp"
+bug-reports:
+  "https://bitbucket.org/jyc/ppx_sexp/issues?status=new&status=open"
+license: "BSD 2-Clause"
+dev-repo: "https://bitbucket.org/jyc/ppx_sexp/src"
```

Comment:
@yallop I have changed the line as requested. Please see https://github.com/ocaml/opam-repository/pull/4662 , which contains an updated version. I made a new pull request because the version has bumped in the meantime.